includes/class-sensei-admin.php

```diff
@@ -734,7 +734,19 @@ private function duplicate_post( $post, $suffix = null, $ignore_course = false )
 			$post_meta = get_post_custom( $post->ID );
 			if ( $post_meta && count( $post_meta ) > 0 ) {
 
-				$ignore_meta = array( '_quiz_lesson', '_quiz_id', '_lesson_quiz', '_lesson_prerequisite' );
+				/**
+				 * Ignored meta fields when duplicating a post.
+				 *
+				 * @hook  sensei_duplicate_content_ignore_meta
```

Comment:
Mismatched hook name

Reply:
Fixed in https://github.com/Automattic/sensei/pull/3830/commits/993543624aee964418da4cad8cad52f3d50e9347